### PR TITLE
[DC-1372] Identifying field suppression

### PR DESCRIPTION
**Pull request by amcgrenera-vumc**

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                for item in schema:
+                    if item['name'] in fields:
+                        if item['mode'] == 'nullable':
+                            value = 'NULL'
+                        elif item['type'] == 'integer':
+                            value = 0
+                        elif item['type'] == 'string':
+                            value = ''
+                        suppression_statement = REPLACE_STRING.render(
+                            suppression_statement=value, field=item['name'])
```
Comment (nishanthpp93) [inline data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression.py:122]: Changing the directory feels a bit off. You can make use of the helper functions in `utils.bq` for this.
```suggestion
          for table in self.affected_tables:
             schema = bq.get_table_schema(table)
             statements = []
             for item in schema:
                 if item.name in fields:
                     if item.mode == 'nullable':
                         value = 'NULL'
                     elif item.field_type == 'integer':
                         value = 0
                     elif item.field_type == 'string':
                         value = ''
                     else:
                         raise RuntimeError(
                             f"Required field {item.name} needs to be integer or string type to be replaced"
                         )
                     suppression_statement = REPLACE_STRING.render(
                         suppression_statement=value, field=item.name)
```

**Review by nishanthpp93 — CHANGES_REQUESTED** [review]
Works well, added a suggestion to avoid changing directories.

File: data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression.py (hunk 1 above)
Comment (amcgrenera-vumc) [reply data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression.py:122]: this is much better, thank you for that suggestion!

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**